plugins/tech-insights-backend/api-report.md

```diff
@@ -100,6 +111,7 @@ export interface RouterOptions<
 > {
   config: Config;
   factChecker?: FactChecker<CheckType, CheckResultType>;
+  factRetrieverEngine: FactRetrieverEngine;
```

Comment:
yeah as you mentioned in your comment this is a breaking change.
Perhaps you can make it optional and throw some warnings if people try to use the new endpoint and `factRetrieverEngine` hasn't been passed?